tests/functional/test_outputter.py

```diff
@@ -0,0 +1,231 @@
+# Aloe - Cucumber runner for Python based on Lettuce and Nose
+# Copyright (C) <2015> Danielle Madeley <danielle@madeley.id.au>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test level 3 outputter
+"""
+
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+# pylint:disable=redefined-builtin, unused-wildcard-import, wildcard-import
+from builtins import *
+# pylint:enable=redefined-builtin, unused-wildcard-import, wildcard-import
+from future import standard_library
+standard_library.install_aliases()
+
+from io import StringIO
+
+from aloe.testing import (
+    FeatureTest,
+    in_directory,
+)
+from mock import patch
+
+
+class TermElement(object):
+    """A colourful element being output by the mock terminal"""
```

Comment:
Okay, I'd like to see a proper docstring for this. I kind of figured it out, but it's not described at all here. For one, can it be a `MockTermElement`?